unstructured/partition/msg.py

```diff
@@ -67,3 +67,32 @@ def build_msg_metadata(msg_obj: msg_parser.MsOxMessage) -> ElementMetadata:
         subject=getattr(msg_obj, "subject", None),
         date=email_date,
     )
+
+
+def extract_attachment_info(
```

Comment:
Should we name this `extract_msg_attachment_info` to deconflict with the `.eml` version if both of these will be public functions?